doc/developers_guide/preparing_a_project_for_native_image_builds.md

```diff
@@ -0,0 +1,11 @@
+# Preparing a project for native-image builds
+
+PK can configure a project to use Graal VM for building platform specific executables for Windows, Mac OS and Linux. Currently, only for amd64 architecture.
+
+## How To
+
+* Add the `native_image` module to your `.project-keeper.yml`.
+* Add integration test that use the native image. This step is important because the native image can fail where the Jar works fine. A typical reason for that is that the native-image does not fully support reflection out of the box.
```

Comment:
```suggestion
* Add integration tests that use the native image. This step is important because the native image can fail where the JAR works fine. A typical reason is that the native-image does not fully support reflection out of the box.
```